src/content/plugins/html-webpack-plugin.md

```diff
@@ -27,7 +27,7 @@ var HtmlWebpackPlugin = require('html-webpack-plugin');
 var webpackConfig = {
   entry: 'index.js',
   output: {
-    path: 'dist',
+    path: __dirname + '/dist',
```

Comment:
Even better would be to use Node's built-in `path.resolve` method:

``` js
path: path.resolve(__dirname, './dist')
```

This would allow the same configuration to work on windows as well.

Reply:
ok, I will change it ,and use path.resolve.

Reply:
Hi, I created another PR, based on my forked repository's master branch.
[1698](https://github.com/webpack/webpack.js.org/pull/1698)
You can close the PR now.

Sorry for the trouble which I caused.
🙃

Reply:
No worries, will do.